caim-org/caim-app
# Upgrade python version, fix parallel migrations, add new email test

Why: I hear python 3.11 is faster/better/stronger, and cus I just wanted to see if it worked. It did.

To test new tests:
`python manage.py test`

to test new python version:

`docker buildx build --platform linux/amd64 .`
then 
`docker run -i ${output of previous command copy it here}`

it should successfully start the server, not encounter errors about missing psycopg2 libraries.
